packages/typography/package.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "@canonical/typography",
+	"module": "index.ts",
```

Comment:
consideration for later: should we publish this as ESM with the calculation function as an export, so that this functionality can be used in other code?